app/views/devise/_links.erb

```diff
@@ -0,0 +1,3 @@
+<% ActiveSupport::Deprecation.warn "Rendering partials devise/_links.erb is deprecated" \
+  "please use devise/shared/_links.erb instead."%>
+<%= render "shared/links" %>
```

Comment:
I don't know how to answer your question. This was auto-generated by devise-i18n-views, which I suppose knows better than I what's the best in this case.